File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.php
class Solution {

    /**
     * @param String $allowed
     * @param String[] $words
     * @return Integer
     */
    function countConsistentStrings($allowed, $words) {
        $allowed_chars = str_split($allowed);
        $counter = count($words);

        foreach($words as $word)
        {
            $letters = str_split($word);
            foreach($letters as $letter)
            {
                $char_not_allowed = !in_array($letter, $allowed_chars);
                if($char_not_allowed)
                {
                    $counter--;
                    break;
                }
            }
        }
        return $counter;
    }
}

File: 1684-count-the-number-of-consistent-strings/1684-count-the-number-of-consistent-strings.cs
public class Solution {
    public int CountConsistentStrings(string allowed, string[] words) {
        return words
            .Count(word => AllLettersAreAllowed(word, allowed));
    }

    private bool AllLettersAreAllowed(string word, string allowed)
    {
        return word.All(letter => allowed.Contains(letter));
    }
}